.github/workflows/TestComposerChecksPlugin.yml

```diff
@@ -0,0 +1,60 @@
+name: ComposerChecksPlugin - Checks Composer checks are okay
```

Comment:
@deviantintegral Did we want this to be its own Action or would it make sense to bundle it in w/ `TestStatic`???

Reply:
Leaving this separate would add ~2m45s each time it runs.

Reply:
I agree, let's bundle it in with TestStatic.

Reply:
Actually, I think this is for testing the plugin code itself. TestStatic (and everything else) will trigger any errors automatically as soon as they run composer install.